diff --git a/7.npmrc b/8.npmrc
index 61a345c..8308caa 100644
--- a/7.npmrc
+++ b/8.npmrc
@@ -13,7 +13,6 @@ modules-cache-max-age=10080
 lockfile=true
 prefer-frozen-lockfile=true
 lockfile-include-tarball-url=false
-use-lockfile-v6=false
 registry=https://registry.npmjs.org/
 <URL>:_authToken=
 <URL>:tokenHelper=
@@ -35,10 +34,10 @@ fetch-retry-factor=10
 fetch-retry-mintimeout=10000
 fetch-retry-maxtimeout=60000
 fetch-timeout=60000
-auto-install-peers=false
-dedupe-peer-dependents=false
+auto-install-peers=true
+dedupe-peer-dependents=true
 strict-peer-dependencies=false
-resolve-peers-from-workspace-root=false
+resolve-peers-from-workspace-root=true
 no-color=auto
 loglevel=info
 use-beta-cli=false
@@ -57,8 +56,9 @@ node-mirror:<releaseDir>=https://nodejs.org/download/<releaseDir>/
 link-workspace-packages=true
 prefer-workspace-packages=false
 shared-workspace-lockfile=true
-save-workspace-protocol=true
+save-workspace-protocol=rolling
 include-workspace-root=false
+ignore-workspace-cycles=false
 use-running-store-server=false
 save-prefix='^'
 tag=latest
@@ -71,6 +71,8 @@ update-notifier=true
 prefer-symlinked-executables=true, when node-linker is set to hoisted and the system is POSIX
 verify-store-integrity=true
 ignore-compatibility-db=false
-resolution-mode=highest
+resolution-mode=lowest-direct
 registry-supports-time-field=false
 extend-node-path=true
+deploy-all-files=false
+dedupe-direct-deps=false
